.github/workflows/build.yaml

```diff
@@ -37,7 +37,7 @@ jobs:
       - name: Build
         run: |
           # Build each provider separately
-          make GOFLAGS="-tags=${{ matrix.provider }}" build
+          make BUILTIN_CLOUD_PROVIDERS="${{ matrix.provider }}" build
```

Comment:
I also needed to update CI.

I think we can also stop running unti tests per cloud provider, and run unit tests for all providers at once. I will raise another PR to do that.
https://github.com/confidential-containers/cloud-api-adaptor/issues/458#issuecomment-1430780931